maps/cogmap.dmm

```diff
@@ -55125,12 +55125,10 @@
 /obj/cable{
 	icon_state = "1-2"
 	},
-/obj/machinery/door/airlock/pyro/command{
-	name = "AI Upload"
-	},
 /obj/mapping_helper/access/ai_upload,
 /obj/mapping_helper/firedoor_spawn,
 /obj/decal/stripe_delivery,
+/obj/machinery/door/airlock/pyro/glass/command,
```

Comment:
name removed?

Reply:
name is automatically applied by area, so its only needed if you want to name a door something that isnt the area name